packages/discovery-react-components/src/components/StructuredQuery/StructuredQuery.tsx

```diff
@@ -40,6 +55,18 @@ const StructuredQuery: FC<StructuredQueryProps> = ({ messages = defaultMessages
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const handleCopyQueryOnClick = () => {
+    const stringifiedSelectionText = document.querySelector(
+      `.${structuredQueryCopyableQueryClass}`
+    )!.textContent;
+    const textArea = document.createElement('textarea');
+    textArea.textContent = stringifiedSelectionText;
+    document.body.appendChild(textArea);
+    textArea.select();
+    document.execCommand('copy');
```

Comment:
Apparently the `execCommand` function isn't available in Safari https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Interact_with_the_clipboard (scroll down a bit or search for the text 'safari'). Maybe we can spin up a tech_debt issue to support this?

Reply:
Ah, good catch. Yeah, I can do that 👍

Reply:
As discussed, I pushed up a change to not include copy functionality (at least until we have clarity on the desired interaction and if it does include this)